app/containers/ItunesProvider/index.js

```diff
@@ -0,0 +1,51 @@
+import React, { memo } from 'react';
+import PropTypes from 'prop-types';
```

Comment:
so we don't really need this index.js. Since we're only drilling the props down in this case. 

so you can delete this index.js and directly import the `ItunesGridContainer` in the routeConfig